### ISSUE TITLE
Archi-Ubuntu.sh has a wrong line endings (CRLF)

### ISSUE THREAD

**Issue by xmstspider** [archimatetool/archi#599]
### Version of Archi, Operating System
4.6 on Linux (Ubuntu 19.10)

### Expected Behaviour
Archi should start normally when executing it with Archi-Ubuntu.sh

### Actual Behaviour
An error is shown and Archi is not starting:
```
$ ./Archi-Ubuntu.sh
bash: ./Archi-Ubuntu.sh: /bin/bash^M: bad interpreter: No such file or directory
```

### Steps to Reproduce the Behaviour
1. Download Archi-Linux64-4.6.0.tgz
2. Extract the archive: tar xvzf Archi-Linux64-4.6.0.tgz
3. Run Archi-Ubuntu.sh

The Bash script has wrong line endings:
```
$ file Archi-Ubuntu.sh
Archi-Ubuntu.sh: Bourne-Again shell script, ASCII text executable, with CRLF line terminators
```

A workaround is to run: dos2unix Archi-Ubuntu-sh

**Comment by Phillipus** [comment]
Hi, this is already be fixed for the next version (4.7 EA 3 dues out soon)

**Comment by Phillipus** [comment]
I'll close this as 4.7 EA3 has the fix.
